### DISCUSSION TITLE
Sudo npm command not found

### DISCUSSION THREAD

**Discussion by kiwistream2015** [mKeRix/room-assistant#1561] [General]
Hi Team, First time user of room assistant. I'm getting the following error when I try to install room assistant.

sudo npm i --global --unsafe-perm room-assistant - After hitting enter I get an error.

sudo npm command not found 

I'm very new please be gentle. 
Any help is appreciated thank you.

**Comment by Turtlepaw** [comment]
Looks like npm isn't installed, try [this](https://stackoverflow.com/a/34558299/15751555)
